lib/repl.js

```diff
@@ -691,11 +691,10 @@ REPLServer.prototype.setPrompt = function setPrompt(prompt) {
   REPLServer.super_.prototype.setPrompt.call(this, prompt);
 };
 
-REPLServer.prototype.turnOffEditorMode = function() {
-  this.editorMode = false;
-  this.setPrompt(this._initialPrompt);
-};
-
+REPLServer.prototype.turnOffEditorMode = util.deprecate(
+  _turnOffEditorMode,
+  'REPLServer.turnOffEditorMode() is deprecated',
+  'DEP00XX');
```

Comment:
The deprecation must be documented as well in the `deprecations.md`.

Reply:
Yes and deprecations are semver-major.